### ISSUE TITLE
Unable to run all Dusk/Browser tests via `php artisan test`

### ISSUE THREAD

**Issue by susanBuck** [laravel/dusk#897]
- Dusk Version: 6.15.0
- Laravel Version: 8.35.1
- PHP Version: 7.4.14
- Database Driver & Version:

### Description:
If I run `php artisan dusk` it finds/runs all my Dusk/Browser tests, reporting back success/failure using standard phpunit output:

![image](https://user-images.githubusercontent.com/237711/115779900-8824f280-a386-11eb-89b3-77aebcd783e2.png)

If I run **an individual Dusk** test via `php artisan test`, the output is much nicer with clearer formatting/color-coding and indication of the exact line where the problem is occuring.

![image](https://user-images.githubusercontent.com/237711/115780029-ae4a9280-a386-11eb-835d-699b1a4fb371.png)

Seeing this, I want to be able to run all my Dusk tests this way. Referencing the docs ([Running tests](https://laravel.com/docs/8.x/testing#running-tests)), I think should be able to do this via `php artisan --testsuite=Browser`, but that yields no tests:

![image](https://user-images.githubusercontent.com/237711/115780381-17caa100-a387-11eb-9f00-154426a72348.png)

**In summary: Is it possible to run all Dusk tests using the _php artisan test_ command with the pretty output?** 

### Steps To Reproduce:
Run `php artisan --testsuite=Browser` - it will not find Browser/Dusk tests, despite the fact that they run when invoking  via `php artisan dusk`.

Observe how `php artisan test [path to Dusk test file]` does work.

**Comment by driesvints** [comment]
Hey @susanBuck, my apologies. I can indeed see you meant this as a feature request. 

The output is actually not controlled by Laravel or Dusk but by Collision. If you'd like Collision to also output Dusk feedback in a nice way it's best that you open up an issue on their repo: https://github.com/nunomaduro/collision

**Comment by SketchNI** [comment]
Sorry to necropost but if @susanBuck is still struggling with this issue, 

navigate to `vendor/laravel/dusk/stubs` and copy `phpunit.xml` to your project root directory and call it `phpunit.dusk.xml` then add the line `printerClass="NunoMaduro\Collision\Adapters\Phpunit\Printer"` to the `<phpunit>` tag in phpunit.dusk.xml.
